### PR TITLE
added palindrome number function and tests to respective files

### PR DESCRIPTION
**Pull request by red-med**

### CODE REVIEW

File: functions/palindrome_number.py
Comment (kimcodes3) [inline functions/palindrome_number.py:1]: Excellent job! My one comment would be to add spaces between operators for better style.

File: functions/palindrome_number.py [hunk 1]
```diff
@@ -1,2 +1,12 @@
 def palindrome_number(x):
-    pass
+    temp = x
+    reverse = 0
+    while(x>0):
+        dig = x%10
```
Comment (hamsurtado) [inline functions/palindrome_number.py:5]: excellent use of mod math trick

File: tests/palindrome_number_test.py
Comment (kimcodes3) [inline tests/palindrome_number_test.py:1]: All these tests are accurate! Add an edge case to it (like 0)

File: tests/palindrome_number_test.py [hunk 2]
```diff
@@ -1,2 +1,34 @@
 import pytest
 from functions.palindrome_number import palindrome_number
+
+def test_returns_true_for_99():
```
Comment (hamsurtado) [inline tests/palindrome_number_test.py:4]: very good tests, excellent

**Review by kimcodes3 — CHANGES_REQUESTED** [review]

**Review by hamsurtado — CHANGES_REQUESTED** [review]
excellent use of math
